resources/tests/org.eclipse.core.tests.resources/src/org/eclipse/core/tests/resources/CharsetTest.java

```diff
@@ -228,7 +227,6 @@ public void testCharsetMoveOnFileMove() throws CoreException {
 	}
 
 	@Test
-	@Ignore("https://github.com/eclipse-platform/eclipse.platform/issues/634")
```

Comment:
This test still fails as intended (the according bug has not been fixed yet). Just removing the `@Ignore` statement currently has no effect, as the test is executed with JUnit 3. It is only used for documentation purposes until I have migrated all the `ResourceTest` implementation to JUnit 4. The actual deactivation for JUnit 3 execution is the `_` prefix of  the method name.